source/cgal/intersections.cc

```diff
@@ -682,25 +678,25 @@ namespace CGALWrappers
       AssertDimension(hexa0.size(), 8);
       AssertDimension(hexa0.size(), hexa1.size());
 
-      std::array<CGALPoint3_inexact, 8> pts_hex0;
-      std::array<CGALPoint3_inexact, 8> pts_hex1;
+      std::array<CGALPoint3_exact, 8> pts_hex0;
```

Comment:
What is the difference? What was the reason to use the inexact one in the first place?

Reply:
The reason is only that inexact kernels are less expensive than exact one (and less robust). I chose them only because I never had issues in my tests/applications. Using exact ones is the only way to have something robust. One could